jaxb-ri/jxc/pom.xml

```diff
@@ -132,12 +138,6 @@
                     </systemPropertyVariables>
                     <forkCount>1</forkCount>
                     <reuseForks>true</reuseForks>
-                    <argLine>
-                        --add-opens org.glassfish.jaxb.jxc/com.sun.tools.jxc.ap=ALL-UNNAMED
-                        --add-opens org.glassfish.jaxb.jxc/com.sun.tools.jxc=ALL-UNNAMED
-                        --module-path "${module.path}"
-                        -Djdk.attach.allowAttachSelf
-                    </argLine>
```

Comment:
why didn't you use `@{argLine}` here instead?

Reply:
Because during `mvn clean install` , it seems that argline is not resolved and it throws an exception. 
So by using argline, it works fine with coverage but not without.
Put argline into properties make it works fine in both cases.